.circleci/config.yml

```diff
@@ -0,0 +1,82 @@
+---
+version: 2.1
+
+orbs:
+    slack: circleci/slack@3.4.2
+
+executors:
+  builder:
+    docker:
+      - image: circleci/php:latest
+  publisher:
+    docker:
+      - image: gcr.io/google.com/cloudsdktool/cloud-sdk:alpine
```

Comment:
hmmmm where did you get this one? I mean https://circleci.com/docs/2.0/google-auth/ uses *google/cloud-sdk*

Reply:
https://cloud.google.com/sdk/docs/downloads-docker